openapi.yaml

```diff
@@ -2088,6 +2164,10 @@ components:
       properties:
         system-security-plan:
           $ref: "#/components/schemas/OSCALPartialUpdateExample"
+    OSCALImplementedRequirement:
+      type: object
+      $ref:
+        "https://raw.githubusercontent.com/usnistgov/OSCAL/main/json/schema/oscal_ssp_schema.json#assembly_oscal-ssp_implemented-requirement"
```

Comment:
This currently needs to reference the `EasyDynamics` fork like the other schema refs (though NIST may be fixing that soon).

You can double check by trying to preview this file in the online Swagger editor.